### PR TITLE
Code runs on a single container, added input capability

### LINKED ISSUE #13: Capture compiler error
Currently, we get something like this on a bad source input:

```
OCI runtime exec failed: exec failed: unable to start container process: exec: "./executed_file.o": stat ./executed_file.o: no such file or directory: unknown
```

The backend should instead capture and return stderr from the compiler, and suppress the internal error message of the backend.

### LINKED ISSUE #14: Preserve new line special character from Fortran program output
See example:

Screenshot from 2022-07-14 08-52-11

### PR DESCRIPTION
**Pull request by ashirrwad**
1. Switched execution method, all the code now runs on a the same container.
2. Added input capability.
3. Fixed #13, Compiler error is being displayed.
4. Fixed #14, Line breaks are being rendered.

### CODE REVIEW

**Comment by ashirrwad** [comment]
@milancurcic @everythingfunctional

**Review by everythingfunctional — APPROVED** [review]
LGTM I tried it out and it seems to work.

File: Backend/app.py [hunk 1]
```diff
@@ -9,53 +9,69 @@
...
+# Starting container
+client = docker.from_env()
+container = client.containers.run("playground-small", tty=True, detach=True)
 
+# Editing the file with code inside editor
+def edit_file(code, input=""):
+    Fortran_file = open("./File.f90", "w+")
+    Fortran_file.write(code)
+    program_input = open("./program_input.txt", "w+")
+    program_input.write(input)
```
Comment (milancurcic) [inline Backend/app.py:23]: Here and elsewhere: should we not clean up after writing, i.e.:

```python
with open("./File.f90", "w+") as f:
    f.write(code)
```

**Review by milancurcic — APPROVED** [review]
Thank you, it looks OK, I think we can go forward pending any outstanding comments.

**Comment by ashirrwad** [comment]
Can we go ahead and merge this?

**Comment by everythingfunctional** [comment]
Sure. I'll open an issue regarding @milancurcic 's comment.

**Pull request merged**